tensor/funcs.go

```diff
@@ -0,0 +1,286 @@
+// Copyright (c) 2024, Cogent Core. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package tensor
+
+import (
+	"fmt"
+	"reflect"
+	"strings"
+
+	"cogentcore.org/core/base/errors"
+	"cogentcore.org/core/base/metadata"
+)
+
+// StringFirstArg should be used to set StringFirst functions
+const StringFirstArg = true
```

Comment:
Again, these bool constants are really not great, nor does it seem like this is a particularly well structured system if you have things like this.

Reply:
These will be updated to a bitflags b/c need more.